browser/src/Services/Learning/Tutorial/Tutorials/TargetsVimPluginTutorial.tsx

```diff
@@ -42,25 +42,33 @@ export class TargetsVimPluginTutorial implements ITutorial {
             new Stages.SetBufferStage([Line1a, Line1b, Line1c]),
             new Stages.SetCursorPositionStage(0, 0),
             new Stages.MoveToGoalStage("Use 'cin(' to move inside the paranthesis", 2, 46),
-            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.MoveToGoalStage("Type 'foo'", 2, 49),
+            new Stages.WaitForModeStage("Press ESC to go back to normal mode", "normal"),
             Stages.combine(
                 null,
                 new Stages.FadeInLineStage(null, 1, Line2a),
                 new Stages.FadeInLineStage(null, 2, Line2b),
-                new Stages.SetBufferStage([Line2a, Line2b, Line2c]),
+                new Stages.FadeInLineStage(null, 3, Line2c),
+                new Stages.SetBufferStage([Line2a, Line2b, Line2c, Line2d]),
             ),
             new Stages.SetCursorPositionStage(0, 7),
             new Stages.MoveToGoalStage("Use 'cin(' to move inside the paranthesis", 0, 40),
-            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.WaitForModeStage("Press ESC to go back to normal mode", "normal"),
             new Stages.SetCursorPositionStage(1, 7),
             new Stages.MoveToGoalStage("Use 'can(' to move around the paranthesis", 1, 39),
-            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.MoveToGoalStage("Type 'bar'", 1, 42),
+            new Stages.WaitForModeStage("Press ESC to go back to normal mode", "normal"),
             new Stages.MoveToGoalStage(
                 "Use 'cil(' to move back inside the last paranthesis",
                 0,
                 40,
             ),
-            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.WaitForModeStage("Press ESC to go back to normal mode", "normal"),
+            new Stages.SetCursorPositionStage(3, 34),
+            new Stages.MoveToGoalStage("Use 'ca(' to select inside the current paranthesis", 3, 32),
```

Comment:
Is this one supposed to be `ci(` instead of `ca(`?

Reply:
I had the user continue to use ca( so that there wasn't a pair of parenthesis afterwards.  This will cause the next ci( to select the same pair of parenthesis.  I could change it and add a step to have the user move the cursor to the right and really drive the point of it selecting the current pair.